src/main/java/me/zilzu/mycoupon/api/controller/CouponController.java

```diff
@@ -43,4 +40,12 @@ public CouponRetrieveListResponse retrieveListCoupons(HttpServletRequest request
         return new CouponRetrieveListResponse("list", requestURI, false, couponListResults);
     }
 
+    @PostMapping("/api/v1/coupons")
+    public CouponCreatedResponse createCoupons(@RequestBody CouponRequest couponRequest) {
```

Comment:
```suggestion
    public CouponCreatedResponse createCoupons(@RequestBody CouponCreationRequest request) {
```

클래스 이름을 좀더 의미가 잘 드러나도록 표현하는게 좋을듯.
CouponRequest 는 어떤 요청인지 알기가 어려움.